apps/testing-javascript/src/lib/lessons.ts

```diff
@@ -20,8 +20,11 @@ export const getLesson = async (
         title
       },
       "stackblitz": resources[@._type == 'stackblitz'][0].openFile,
-      "videoResourceId": resources[@->._type == 'videoResource'][0]->_id,
-      "transcript": resources[@->._type == 'videoResource'][0]-> castingwords.transcript,
+      ...resources[@->._type == 'videoResource'][0]-> {
+        "videoResourceId": _id,
+        "durationInSeconds": duration,
+        "transcript": castingwords.transcript
+      },
```

Comment:
combine duration and other `videoResource` attributes into a single "query", otherwise, it has to load the videoResource three different times